feconf.py

```diff
@@ -520,6 +520,12 @@ def get_empty_ratings():
 UPDATE_TYPE_COLLECTION_COMMIT = 'collection_commit'
 UPDATE_TYPE_FEEDBACK_MESSAGE = 'feedback_thread'
 
+# User query status.
```

Comment:
Just a note: please either keep the documentation for your "state diagram" here (or, later, put it in the part of the code that handles the transition logic). It's helpful context.